docs/start/authentication/customer-login.mdx

```diff
@@ -42,6 +42,7 @@ JWT is an industry standard ([RFC 7519](https://tools.ietf.org/html/rfc7519)) fo
 | `customer_id` | integer | The ID of the shopper who is signing in.|
 | `redirect_to` | string | Optional field containing a relative path for the shopper's destination after sign-in. Defaults to `/account.php`. |
 | `request_ip` | string | Optional field containing the expected IP address for the request. If provided, BigCommerce will check that it matches the browser trying to sign in.|
+| `merge_persistent_cart` | boolean | Optional field to specify whether or not to merge carts during customer token login. This feature depends on the [Persistent Cart](https://support.bigcommerce.com/s/article/Persistent-Cart) setting being enabled on your store. The default value is `false`.|
```

Comment:
An optional field to specify whether or not to merge carts during customer token login. This feature depends on you enabling the [Persistent Cart](https://support.bigcommerce.com/s/article/Persistent-Cart) setting on your store.